GPU/GPUTracking/Merger/GPUTPCGMO2Output.cxx

```diff
@@ -113,6 +114,7 @@ GPUdii() void GPUTPCGMO2Output::Thread<GPUTPCGMO2Output::output>(int nBlocks, in
   const unsigned int flagsRequired = getFlagsRequired(merger.Param().rec);
   TrackTPC* outputTracks = merger.OutputTracksTPCO2();
   unsigned int* clusRefs = merger.OutputClusRefsTPCO2();
+  PIDResponse pidResponse{};
```

Comment:
Could you move the instanciation of this class also into the if branch. No reason to always have it, and it will simplify compiler optimizations.